### DISCUSSION TITLE
求大佬看看

### DISCUSSION THREAD

**Discussion by ldfgood** [wzyonggege/statistical-learning-method#8] [Q&A] [answered]
import pandas as pd
import numpy as np
from sklearn.datasets import load_iris
import matplotlib.pyplot as plt
%matplotlib inline
里面的这个%我看不懂，而且编译器也报错，我应该怎么改

**Comment by www5226448** [comment, marked as answer]
删掉这个，这个是notebook里用的的magic code

**Reply by ldfgood** [reply]
谢谢大佬
